<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_Accounts</class>
 <widget class="QWidget" name="Settings_Accounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true" extracomment="not translatable because not shown to the user">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,0,0">
     <item>
      <widget class="QButton" name="installFromFileBtn" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter by Capability:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="accountsFilterCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="AccountListView" name="accountsView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QButton</class>
   <extends>QWidget</extends>
   <header>thirdparty/Qocoa/qbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccountListView</class>
   <extends>QListView</extends>
   <header>widgets/AccountListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
